base/src/user_model/history.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(deprecated)]
```

Comment:
Please remove all deprecated functions. We really don't need them. At this point it is ok to break this.

Reply:
Will do!